docs/how-to-add-a-new-rule.md

```diff
@@ -46,29 +45,31 @@ and which probe code to inject, etc.:
 ]
 ```
 
-- `ImportPath`: The import path of the package that contains the function to be instrumented.
-- `Function`: The name of the function to be instrumented.
-- `OnEnter`: The name of the function to be called when the instrumented function is called.
-- `Path`: The path to the directory containing the probe code.
+- `ImportPath`: The import path of the package that contains the function
+- `Function`: The function to instrument.
+- `OnEnter`: The hook code
+- `Path`: Directory containing the hook code.
 
-There are many other fields that can be specified in the rule, such as 
-`OnExit`, `Order`, etc. You can refer to [this document](rule_def.md) for more information.
+Additional fields like `OnExit` and `Order` can also be specified. Refer to [the documentation](rule_def.md) for details.
 
-## 3. Verify the rule
-To make sure the rule is working as expected, we can write a simple demo program to test it:
+## 3. Verify the Rule
+Test the rule with a simple program:
 
 ```bash
-$ mkdir setenv && cd setenv
-$ go mod init setenv
+$ mkdir demo && cd demo
```

Comment:
May be another name like `setenv-demo` is better?

Reply:
Make sense.